tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-envlist=pycodestyle,pylint,yamllint
+envlist=pycodestyle,pylint,yamllint,policy
```

Comment:
Is this change related to the Python 3.9 change?

Reply:
Yes, we were not able to run the policy check anymore with Ubuntu18.04/Python3.7. I will mention this change in the PR description.

Reply:
Ok, this raise a problem with community.general. I will isolate this change in a new PR.

Reply:
I'm curious, how does this change have any effect on community.general?

Reply:
This was the error:

```
TASK [Fail if policy is too large] ***********************************************************************************************************************************************************************************************************
[WARNING]: an unexpected error occurred during Jinja2 environment setup: unable to locate collection community.general
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while templating '{{ policies | json_query(_query) | list }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: template error while templating string: unable to locate collection community.general. String: {{ policies | json_query(_query) | list }}"}
```

See: https://github.com/mattclay/aws-terminator/pull/250